app/parse.py

```diff
@@ -8,5 +13,18 @@ class Course:
     duration: str
 
 
+def parse_single_course(course_soup: Tag) -> Course:
+    name = course_soup.select_one("a.ProfessionCard_title__Zq5ZY > h3").text
```

Comment:
The CSS selector `a.ProfessionCard_title__Zq5ZY > h3` may not be reliable if the class names are dynamically generated or change over time. Consider using more stable selectors or verifying the current structure of the webpage.